msticpy/data/queries/kql_sent_azure.yaml

```diff
@@ -123,4 +123,20 @@ sources:
       resource:
         description: ResourceId or substring
         type: str
+  morph_signins:
```

Comment:
Seems like an odd name for a pretty generic query - maybe AADSigninsWithGeogLoc ?